services/ansibler/server/server.go

```diff
@@ -97,7 +99,7 @@ func (*server) TeardownLoadBalancers(ctx context.Context, req *pb.TeardownLBRequ
 	for _, lb := range req.DeletedState.LoadBalancerClusters {
 		np, ok := k8sNodepools[lb.TargetedK8S]
 		if !ok {
-			log.Error().Msgf("LoadBalancer %s has not found a target k8s cluster %s", lb.ClusterInfo.Name, lb.TargetedK8S)
+			log.Info().Msgf("LoadBalancer %s has not found a target k8s cluster %s", lb.ClusterInfo.Name, lb.TargetedK8S)
```

Comment:
Isn't this considered an error/warning? If we have a Loadbalancer in a state which does not target any Kubernetes cluster, it should be caught by the frontend, thus if we detect it here, something went wrong.